src/main/kotlin/com/server/jobboxapp/gateway/BlogGateway.kt

```diff
@@ -0,0 +1,32 @@
+package com.server.jobboxapp.gateway
+
+import com.server.jobboxapp.entity.*
+import com.server.jobboxapp.service.BlogService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/blog")
+class BlogGateway(
+    private val blogService: BlogService
+) {
+    @GetMapping
+    @CrossOrigin(origins = ["*"])
```

Comment:
Cross powinien być globalny. Narazie możemy zostawić otwarte API na wszystko ([*]), ale potem trzeba będzie to filtrować